include/lsst/afw/math/PixelScaleBoundedField.h

```diff
@@ -44,7 +44,8 @@ namespace math {
  *
  * Typically used to move an image or source flux between surface brightness and fluence space.
  */
-class PixelScaleBoundedField : public BoundedField {
+class [[deprecated("Replaced by PixelAreaBoundedField; will be removed on DM-22192.")]]
```

Comment:
I thought that deprecation messages should say what release they'll be deprecated before (e.g., prior to 20.0)